addons/website/views/snippets/s_popup.xml

```diff
@@ -40,23 +41,26 @@
         <div data-js="SnippetPopup" data-selector=".s_popup" data-target=".s_popup_main">
             <we-select string="Position" class="o_we_inline">
                 <we-button data-set-layout="modal" data-name="position_centered">Center</we-button>
-                <we-button data-set-layout="fixed">Bottom</we-button>
+                <we-button data-set-layout="fixed" data-name="position_bottom">Bottom</we-button>
             </we-select>
             <we-select string="Size" data-apply-to=".s_popup_frame" class="o_we_inline">
                 <we-button data-select-class="modal-sm">Small</we-button>
-                <we-button data-select-class="modal-md">Medium</we-button>
+                <we-button data-select-class="modal-md" data-name="s_popup_size_md">Medium</we-button>
                 <we-button data-select-class="modal-lg">Large</we-button>
                 <we-button data-select-class="modal-xl">Big</we-button>
-                <we-button data-select-class="s_popup_full">Full</we-button>
+                <we-button data-select-class="s_popup_full" data-name="s_popup_size_full">Full</we-button>
             </we-select>
             <we-colorpicker string="Backdrop Color" data-select-style="true"
                 data-css-property="background-color" data-color-prefix="bg-" data-dependencies="position_centered"/>
+        </div>
+        <div data-js="SnippetPopup" data-selector=".s_popup" data-exclude="#website_cookies_bar" data-target=".s_popup_main">
```

Comment:
I feel like this should be done in JS with compute visibility, as I will need to also hide 'Position' which is the first one.
If I keep doing it this way, we will have to split again the data-js. I wrote it like that at first but I think it was not working since it is not its own snippet option (splitted data-JS).

Will investigate..

Plus not very modular, if another whatever code need to use this and hide other widget options (most likely wont happen), it won't be able to do it in XML anyway.

Reply:
Not sure to understand tell me if it is still an up-to-date issue ?